runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -333,6 +333,16 @@ else if (mouseInHighlightBox)
 				drawRectangle(graphics, itemHighlightBox, topItem && mouseInHighlightBox ? Color.GREEN : color, highlighted != null, false);
 			}
 
+			// Draw outline if option toggled
```

Comment:
I feel this comment can be removed, it's pretty clear what this code block does.